mungegithub/features/aliases.go

```diff
@@ -72,33 +72,28 @@ func (a *Aliases) Name() string {
 
 // Initialize will initialize the feature.
 func (a *Aliases) Initialize(config *github.Config) error {
-	a.data = &aliasData{
-		AliasMap: map[string][]string{},
-	}
+	a.data = &aliasData{}
+	a.aliasReader = a
 
-	if len(a.aliasFile) == 0 {
-		return nil
+	if len(a.aliasFile) > 0 {
+		a.IsEnabled = true
```

Comment:
wouldn't it make more sense to return an error if this munger is enabled and there's no aliasFile?